validation.js

```diff
@@ -0,0 +1,19 @@
+const emailInput = document.getElementById('email');
+const form = document.getElementById('contact-form');
+const validateMessage = document.getElementById('validate');
+function validate() {
+  const email = emailInput.value.trim();
+  const isLowercase = email.toLowerCase();
+  if (email !== isLowercase) {
+    validateMessage.innerText = 'Please enter a lowercase email address';
+    validateMessage.style.color = 'red';
+    return false;
+  }
+  return true;
+}
+
+form.addEventListener('submit', (event) => {
+  if (validate() === false) {
+    event.preventDefault();
+  }
```

Comment:
- [ ] Please remove the conditional clause here as even if the form inputs are not valid, it sends the request to the URL. We should see the errors before submitting the form. 
![image](https://user-images.githubusercontent.com/91486791/232086652-7edae16e-1df7-4bd7-8972-5a0debc0b72d.png)